addons/mail/static/src/owl/components/composer_text_input/composer_text_input.scss

```diff
@@ -40,19 +63,4 @@
     > .note-resizebar {
         display: none;
     }
-}
-
-.o_ComposerTextInput_mentionMenuItem {
-    text-overflow: ellipsis;
-
-    .o_PartnerImStatusIcon {
-        display: inline-flex;
-        margin-right: 5px;
-    }
-
-    > .o_extra {
-        text-overflow: ellipsis;
-        font-style: italic;
-        color: gray('500');
-    }
```

Comment:
take opportunity to add new line at end of file

Reply:
Done